client/src/app/components/about/about.component.css

```diff
@@ -16,3 +16,21 @@
   max-width: 70%;
 }
 
+h4 {
+  color: #384163;
+}
+
+#hover {
```

Comment:
'#' means id which is supposed be unique on entire  page, it seems you intend to use it multiple times - this is not correct, make it a class